### PR TITLE
Docs 7968 salesforce 10 1 0 dh

### PR DESCRIPTION
**Pull request by dhirsch1**

### CODE REVIEW

File: modules/ROOT/pages/salesforce/v10.1/salesforce-connector-101-config-topics.adoc [hunk 1]
```diff
@@ -291,6 +291,49 @@ output application/java
...
++
+The free version of Object Store v2 has a limit of 10 transactions per second.
+
+For more information about object store versions, see https://docs.mulesoft.com/object-store/#object-store-notes[Object Store Notes].
+
+=== Replay Topic and Replay Channel Listener Operations
+
+The Replay topic and Replay channel listener operations have the option to continue from the last replay ID they received before restarting the application.
+
+When a Mule app starts for the first time, the connector creates an object store that is used to save data related to messages that processed successfully or that failed:
```
Comment (alexandruatomei9) [inline modules/ROOT/pages/salesforce/v10.1/salesforce-connector-101-config-topics.adoc]: that WERE processed ?

File: modules/ROOT/pages/salesforce/v10.1/salesforce-connector-101-config-topics.adoc [hunk 2]
```diff
@@ -291,6 +291,49 @@ output application/java
...
+
+* For each message that processed successfully, the information stored consists of a a replay ID associated with the message.
```
Comment (alexandruatomei9) [inline modules/ROOT/pages/salesforce/v10.1/salesforce-connector-101-config-topics.adoc]: that WAS processed

**Review by alexandruatomei9 — APPROVED** [review]

File: modules/ROOT/pages/salesforce/salesforce-connector.adoc [hunk 3]
```diff
@@ -12,12 +12,21 @@ This documentation set consists of the following version documents:
 [%header,cols="40a,60a"]
 |===
 |Version Documents |Updates in This Release
+
+|xref:salesforce/v10.1/salesforce-connector-101.adoc[Salesforce Connector v10.1] +
+xref:salesforce/v10.1/salesforce-connector-reference-101.adoc[Salesforce Connector Reference v10.1]
+
+| * Added support for API v48.0
+* Added support for subscribing to these newly-added platform events: FOStatusChangedEvent, OrderSummaryCreatedEvent, and OrderSumStatusChangedEvent
+* Added support for using the new RecordVisibilityContext clause when using the WITH filteringExpression in a query
```
Comment (kevintroller) [inline modules/ROOT/pages/salesforce/salesforce-connector.adoc:21]: Is WITH upper case intended?

File: modules/ROOT/pages/salesforce/v10.1/salesforce-connector-101-examples.adoc [hunk 4]
```diff
@@ -0,0 +1,380 @@
...
+endif::[]
+:imagesdir: ../../assets/images/
+
+[[accandtrans]]
+== Example: Accept and Transform Data
+
+The Mule flow sets up:
+
+* HTTP Listener: Accepts data from HTTP requests.
+* Transform Message: Transforms data structure and format to produce the output that the Salesforce connector expects.
```
Comment (kevintroller) [inline modules/ROOT/pages/salesforce/v10.1/salesforce-connector-101-examples.adoc]: Salesforce connector > Salesforce Connector

File: modules/ROOT/pages/salesforce/v10.1/salesforce-connector-101-examples.adoc [hunk 5]
```diff
@@ -0,0 +1,380 @@
...
+* Salesforce connector:
```
Comment (kevintroller) [inline modules/ROOT/pages/salesforce/v10.1/salesforce-connector-101-examples.adoc]: ditto

File: modules/ROOT/pages/salesforce/v10.1/salesforce-connector-101-examples.adoc [hunk 6]
```diff
@@ -0,0 +1,380 @@
...
++
+** (Outbound) Connects with Salesforce and performs an operation to push data to Salesforce.
+** Triggers a flow according to the configuration.
+** Connects with Salesforce, and returns an `InputStream` with the query results.
+* Transform Message: Transforms a data structure and format to produce the output that the File connector endpoint expects.
+* File connector: Records data in a file, such as a CSV and saves it to a user-defined directory or location.
```
Comment (kevintroller) [inline modules/ROOT/pages/salesforce/v10.1/salesforce-connector-101-examples.adoc]: File Connector

File: modules/ROOT/pages/salesforce/v10.1/salesforce-connector-101-examples.adoc [hunk 7]
```diff
@@ -0,0 +1,380 @@
...
+----
+
+[[parentchild]]
+== Example: Create or Update an Object With Parent Child Relationships
+
+A Salesforce object can have standard or custom relationships between objects.
+
+The relationships between the objects are usually one-to-many parent child relationships, but can be any link between two objects residing in Salesforce.
+
+Creating or altering objects with relationships is challenging. This section shows how to perform an upsert for an object using the Salesforce connector.
```
Comment (kevintroller) [inline modules/ROOT/pages/salesforce/v10.1/salesforce-connector-101-examples.adoc]: ditto

File: modules/ROOT/pages/salesforce/v10.1/salesforce-connector-101-examples.adoc [hunk 8]
```diff
@@ -0,0 +1,380 @@
...
+
+Create a structure in Salesforce for this relationship.
+
+This example assumes two custom types: `MyCustomObject` and `MyOtherCustomObject`.
+
+MyCustomObject must hold a relationship to a MyOtherCustomObject. When upserting a `MyCustomObject`, the POJO sent as input to the Salesforce connector looks like this:
```
Comment (kevintroller) [inline modules/ROOT/pages/salesforce/v10.1/salesforce-connector-101-examples.adoc]: ditto

File: modules/ROOT/pages/salesforce/v10.1/salesforce-connector-101.adoc [hunk 9]
```diff
@@ -0,0 +1,205 @@
...
+* https://help.salesforce.com/articleView?id=user_security_token.htm[Salesforce security token]
++
+You can receive a new security token by email if you run *Reset Security Token* through *My Personal Information* from the Salesforce Setup pages.
+* Consumer key and secret (available in your Salesforce developer account)
++
+Required if you are using the OAuth Authentication type
++
+* Namespace and schema location
++
+If you plan to create the XML for your Mule app, include the correct namespace and schema location in your XML file. Studio adds this information to the XML file automatically when you add the Salesforce connector to a flow in a Mule app.
```
Comment (kevintroller) [inline modules/ROOT/pages/salesforce/v10.1/salesforce-connector-101.adoc]: Salesforce Connector

**Review by kevintroller — APPROVED** [review]
Approved. Minor comments

File: modules/ROOT/pages/salesforce/salesforce-connector.adoc (hunk 3 above)
Comment (dhirsch1) [reply modules/ROOT/pages/salesforce/salesforce-connector.adoc:21]: Yes

**Pull request merged**